src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -229,6 +229,11 @@ When set to `true`, it enables export of attributes corresponding to
 `exception.stacktrace` are defined in
 [specification](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/exceptions/exceptions-logs.md#attributes).
 
+* `OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_CATEGORY_AND_EVENT_LOG_ATTRIBUTES`
+
+When set to `true`, it enables export of `LogRecord.EventId` as `event.id` and
```

Comment:
nit: consider - When set to true, it exports LogRecord.EventId as event.id, LogRecord.CategoryName as dotnet.ilogger.category, and event.name.

Reply:
It would export `LogRecord.EventId.Id` as `event.id` and `LogRecord.EventId.Name` as `event.name`.